src/lib.rs

```diff
@@ -19,13 +22,16 @@ mod flexbox;
 mod forest;
 mod sys;
 
+pub use crate::node::Sprawl;
+
 #[cfg(feature = "std")]
 use core::fmt::{Display, Formatter, Result};
 
-pub use crate::node::Sprawl;
-
+/// An error that can occur when performing layout
+#[non_exhaustive]
```

Comment:
This is a breaking change, right? We will need to remember this for the changelog.

Reply:
```suggestion
```

I'm just going to revert.